F1/F1/Utilities/Constants.swift

```diff
@@ -12,6 +12,7 @@ struct Identifiers {
     static let DriverTableViewCell = "DriverTableViewCell"
     static let ConstructorTableViewCell = "ConstructorTableViewCell"
     static let ConstructorResultsIdentifier = "ConstructorResultHeader"
+    static let RaceScheduleIndentifier = "RaceScheduleTableViewCell"
     static let showDriverSegue = "showDriverSegue"
```

Comment:
still need to make this change :) 
static let showDriverSegue = "ShowDriverSegue"
static let showConstructorSegue = "ShowConstructorSegue"

Keep them starting with a caps since all others start with a caps :)